### ISSUE TITLE
slack_channel parameter is not working

### ISSUE THREAD

**Issue by jiridudekusy** [RasaHQ/rasa_core#371]
**Rasa Core version**:
0.8.5
**Python version**: 
3.6.5
**Operating system** (windows, osx, ...):
Ubuntu (docker image python)
**Issue**:
slack_channel parameter from slack channel configuration is not used for output messages. In fact in source code (https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/channels/slack.py) it is not used at all for anything useful.

I suppose that slack_channel will be used as a recipient for all outgoing messages.

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by tmbo** [comment]
that's right, it currently only directly responds to user messages - seems to be a bug. should be possible to always send to a channel, right @ricwo?

**Comment by ricwo** [comment]
Thanks @jiridudekusy, this is indeed a bug if we want to support always sending messages to a static recipient (channel or user). 

`self.slack_channel` should be passed to the `SlackBot.__init__()` method. The `send_` methods should check for the presence of that member, and use it in the `api_call` if present instead of `recipient_id`

**Comment by melbit-michaelw** [comment]
Hi, 

This doesn't appear to be working for me.. Even when a slack_channel is configured in my credentials.yml, the bot responds directly to users instead. 

In slack.py, the process_message function makes this call:

out_channel = SlackBot(self.slack_token)

which means when it creates the SlackBot, it isn't setting the output channel, and thus self.slack_channel in the SlackBot class is always None, unless I've missed something ?

**Comment by akelad** [comment]
@melbit-michaelw please create a new issue
